main.py

```diff
@@ -2,9 +2,49 @@
 
 from db.models import Race, Skill, Player, Guild
 
+import json
+
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+
+    for key, value in players.items():
+        check_race = Race.objects.filter(name=value["race"]["name"])
+        if not check_race.exists():
+            Race.objects.create(
+                name=value["race"]["name"],
+                description=value["race"]["description"]
+            )
+        race = Race.objects.get(name=value["race"]["name"])
```

Comment:
Consider using `get_or_create` for Race to simplify the logic and reduce the number of database queries. This method can handle both checking for existence and creating the object if it doesn't exist.